builds/build-release.yml

```diff
@@ -10,7 +10,7 @@ variables:
   versionMinor: 1
   versionMajorMinor: '$(versionMajor).$(versionMinor)'  # This variable is only used for the counter
   versionPatch: $[counter(variables['versionMajorMinor'], 0)] # This will reset when we bump either major or minor version
-  versionSuffix: 'alpha'
+  versionSuffix: 'preview'
```

Comment:
Maybe update the readme also?

Reply:
and also update the sample project that references the alpha package after the preview package gets released

Reply:
I added #75 to track this. I believe the new version will be 0.1.2-preview but I'll need to run the release first to verify. To avoid potentially having to change the version again I'll do this in a separate PR.